pyth/plugins/rtf15/reader.py

```diff
@@ -139,57 +149,57 @@ def parse(self):
                 # inside groups we don't care about anyway
                 continue
 
-            elif next == '\\':
+            elif nextbyte == b'\\':
                 control, digits = self.getControl()
                 self.group.handle(control, digits)
             else:
-                self.group.char(next)
+                self.group.char(nextbyte)  # within-group text
 
 
     def getControl(self):
-        chars = []
-        digits = []
-        current = chars
-        first = True
+        chars = []  # for output 1
```

Comment:
These comments could do with being either improved or removed. As-is, even I'm confused by them.